contracts/DCNT721A.sol

```diff
@@ -96,13 +127,81 @@ contract DCNT721A is ERC721A, Initializable, Ownable, Splits {
     }
 
     _safeMint(msg.sender, numberOfTokens);
-    for (uint256 i = 0; i < numberOfTokens; i++) {
-      emit Minted(msg.sender, mintIndex++);
+    unchecked {
+      for (uint256 i = 0; i < numberOfTokens; i++) {
+        emit Minted(msg.sender, mintIndex++);
+      }
+    }
+  }
+
+  // allows the owner to "airdrop" users an NFT
+  function mintAirdrop(address[] calldata recipients) external onlyOwner {
```

Comment:
```suggestion
  function airdrop(address[] calldata recipients) external onlyOwner {
```

personally, I think the `mint` is implied and if someone is looking for the `airdrop` functionality on Etherscan they're going to look at the top in `a` rather than in `m` but as I'm typing this I could see the argument to group all `mint` functions together as well.

Reply:
perfectly fine with this change, but need to update the new unit tests, otherwise i would have added this to batch.